.github/scripts/contentful/create.sh

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env bash
 
-if [[ "ON_BRANCH_ENV" == "False" ]]; then
+if [[ "ON_BRANCH_ENV" == "false" ]]; then
   echo "No branch Environment required; not deleting environment."
```

Comment:
this message is confusing to me.

The variable is 'ON_BRANCH_ENV' i.e. whether we are on a dev branch. 

And this message is printed when we are not on dev.

Reply:
Yeah agree, this should probably be something like "No branch environment to delete."

Reply:
I have made this change @markcsims.